src/main/java/com/uber/jenkins/phabricator/uberalls/UberallsClient.java

```diff
@@ -67,13 +66,12 @@ public String getBaseURL() {
         return this.baseURL;
     }
 
-    public CodeCoverageMetrics getParentCoverage(Differential differential) {
+    public CodeCoverageMetrics getParentCoverage(String sha) {
+        if (sha == null) {
```

Comment:
Yup good idea. Was just preserving existing behavior, but you're right that that's more correct. Will fix in a future PR